File: 2.0/java/calls/getCall.java
import com.callfire.api.client.CallfireClient;
import com.callfire.api.client.api.callstexts.model.Call;

class ApiClientSample {
    public static void main(String[] args) {
        CallfireClient client = new CallfireClient("api_login", "api_password");
        Call call = client.callsApi().get(617067920003L);
    }
}

File: 2.0/csharp/calls/getCall.cs
using CallfireApiClient;
using CallfireApiClient.Api.CallsTexts.Model;

public class ApiClientSample
{
    public static void Main(string[] args)
    {
        var client = new CallfireClient("api_login", "api_password");
        Call call = client.CallsApi.Get(379506003, "fromNumber,toNumber,modified,finalCallResult");
    }
}
